pylint/extensions/code_style.py

```diff
@@ -28,7 +28,7 @@ class CodeStyleChecker(BaseChecker):
        i.e. detect a common issue or improve performance
        => it should probably be part of the core checker classes
     2. Is it something that would improve code consistency,
-       maybe because it's slightly better with regards to performance
+       maybe because it's slightly better with regard to performance
```

Comment:
I think it should be `regards` actually, but I'm doubting myself.

Reply:
from google: regard is the original "correct" one but regards is catching up quickly, so not a big deal either way.